examples/fungible-token/io/Cargo.toml

```
@@ -4,7 +4,6 @@ version = "0.1.4"
 authors.workspace = true
 edition.workspace = true
 license.workspace = true
-workspace = "../../../"
 
 [dependencies]
 gstd = { workspace = true }
```

Seems you didn't totally unify the toml style 
```suggestion
gstd.workspace = true
```

---

And introduce similar feature gate for all programs
Furthermore you can't split them all into lib.rs and wasm.rs, cause some of our demo doesnt compile correct but don't show up on CI and tests due to cargo workspace issues

---

ref #3069